packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st

```diff
@@ -2,18 +2,13 @@ tests
 testHashchangesAfterMetaClassInformationChange
 	| originalComment  testClass hashBeforeChange hashAfterChange |
 
-	CCESession default scope: #( 'Signals' ).
-	
-	testClass := (PackageInfo named: 'Signals') classes first.
-	
+	CCESession default scope: #( 'CollabCodeEdit-Tests' ).
+	testClass := Environment default classNamed: 'CCETestDummy'.
 	originalComment := testClass comment.
- 
-	hashBeforeChange := packageManager hashPackages: #( 'Signals' ).
+	[ hashBeforeChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
 	
 	testClass comment: 'This comment was changed by packageManagerTest'.
+	hashAfterChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
 	
- 	hashAfterChange := packageManager hashPackages: #( 'Signals' ).
-	
-	self deny: hashBeforeChange equals: hashAfterChange.
-	
-	testClass comment: originalComment
\ No newline at end of file
+	self deny: hashBeforeChange = hashAfterChange. ]
+	ensure: [testClass comment: originalComment]
```

Comment:
The current indentation makes it unclear that `ensure:` belongs to the preceding block.